.azure-pipelines/azure-pipelines-osx.yml

```diff
@@ -8,38 +8,11 @@ jobs:
     vmImage: macOS-11
   strategy:
     matrix:
-      osx_64_python3.10.____cpython:
-        CONFIG: osx_64_python3.10.____cpython
+      osx_64_:
+        CONFIG: osx_64_
         UPLOAD_PACKAGES: 'True'
-      osx_64_python3.11.____cpython:
-        CONFIG: osx_64_python3.11.____cpython
-        UPLOAD_PACKAGES: 'True'
-      osx_64_python3.12.____cpython:
-        CONFIG: osx_64_python3.12.____cpython
-        UPLOAD_PACKAGES: 'True'
-      osx_64_python3.8.____cpython:
-        CONFIG: osx_64_python3.8.____cpython
-        UPLOAD_PACKAGES: 'True'
-      osx_64_python3.9.____73_pypy:
-        CONFIG: osx_64_python3.9.____73_pypy
-        UPLOAD_PACKAGES: 'True'
-      osx_64_python3.9.____cpython:
-        CONFIG: osx_64_python3.9.____cpython
-        UPLOAD_PACKAGES: 'True'
-      osx_arm64_python3.10.____cpython:
```

Comment:
Why you removed arm?

Reply:
These changes have not removed these platforms at all. See the new set of ci support files in the PR.